masterlist.yaml

```diff
@@ -1634,9 +1634,8 @@ plugins:
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/24941/' ]
     group: *fixesGroup
     msg:
-      - <<: *alreadyInOrFixedByX
-        subs: [ 'Unofficial Skyrim Special Edition Patch' ]
-        condition: 'active("Unofficial Skyrim Special Edition Patch.esp")'
+      - <<: *useInstead
+        subs: [ '[Unofficial Skyrim Special Edition Patch](https://www.nexusmods.com/skyrimspecialedition/mods/266)' ]
```

Comment:
I don't mind adding this, but It's unlikely anyone not using the USSEP is going to switch or stop using the file.
A condition needs to be added as well, since it's a warning message.

Reply:
Fair point, I just figured that since the mod can break the quest, it's worth adding a message even for the handful of people it'd be useful to. I've also added a condition, sorry about that.

Reply:
Okay, there's no harm in adding it.